### PR TITLE
Issue#70

### LINKED ISSUE #70: Rename `[ASTypeKey]` attribute
Rename the attribute to something else. Reasoning:
1. It defines an ActivityPub type, not Activity*Streams*.
2. May be used to track metadata at a future point (such as #39 and #36).

Proposed name: `APTypeAttribute` / `[APType]`

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
fix #70

Hi [warriordog](https://github.com/warriordog),
This is my PR  for this issue, please check it out

### CODE REVIEW

**Comment by warriordog** [comment]
@viktoriussuwandi Hello, thank you for the PR! Are some of the changes missing? Many of the types are not updated, and the renamed `APTypeAttribute` class isn't in any of the commits.

**Comment by viktoriussuwandi** [comment]
no worries @warriordog, I'll do it step by step untill all files covered, thanks for your advice :)

**Comment by viktoriussuwandi** [comment]
Hi @warriordog, I think it is ready to review, just lemme know if there is still need to be updated

**Comment by warriordog** [comment]
@viktoriussuwandi thank you! Reviewing now

File: ActivityPub.Types/Json/ASTypeKeyAttribute.cs [hunk 1]
```diff
@@ -7,13 +7,13 @@ namespace ActivityPub.Types.Json;
 
 [AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = true)]
 [MeansImplicitUse]
-public sealed class ASTypeKeyAttribute : Attribute
+public sealed class ASTypeAttribute : Attribute
```
Comment (warriordog) [inline ActivityPub.Types/Json/ASTypeKeyAttribute.cs:10]: Shouldn't this be `APTypeAttribute`?

**Review by warriordog — CHANGES_REQUESTED** [review]
I left a couple of comments. Also, this needs to be rebased onto the latest changes. There's a lot of them, so please let me know if you'd like help with that!

File: ActivityPub.Types/Json/ASTypeKeyAttribute.cs (hunk 1 above)
Comment (viktoriussuwandi) [reply ActivityPub.Types/Json/ASTypeKeyAttribute.cs]: Already updated @warriordog

File: ActivityPub.Types/Json/ASTypeKeyAttribute.cs
[the remark this answers is not in the document]
Comment (viktoriussuwandi) [reply ActivityPub.Types/Json/ASTypeKeyAttribute.cs:1]: already renamed the file @warriordog

**Comment by warriordog** [comment]
@viktoriussuwandi this looks good now, just needs a rebase. I can do that part myself, if you'd like. Just let me know

**Comment by viktoriussuwandi** [comment]
Thanks for the opportunity @warriordog, I leave the rest (rebase part) for you :)

**Comment by warriordog** [comment]
Merged in #107, for technical reasons

**Pull request closed without merging**